assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.js

```diff
@@ -0,0 +1,275 @@
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { Fragment, useCallback, useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useDispatch, useSelect } from 'googlesitekit-data';
+import whenActive from '../../../../../util/when-active';
+import { CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import {
+	CORE_NOTIFICATIONS,
+	NOTIFICATION_GROUPS,
+} from '../../../../../googlesitekit/notifications/datastore/constants';
+import { AUDIENCE_SEGMENTATION_SETUP_FORM } from '../../../datastore/constants';
+import { SETTINGS_VISITOR_GROUPS_SETUP_SUCCESS_NOTIFICATION } from '../settings/SettingsCardVisitorGroups/SetupSuccess';
+import useViewContext from '../../../../../hooks/useViewContext';
+import {
+	AdminMenuTooltip,
+	useShowTooltip,
+	useTooltipState,
+} from '../../../../../components/AdminMenuTooltip';
+import { withWidgetComponentProps } from '../../../../../googlesitekit/widgets/util';
+import { trackEvent, WEEK_IN_SECONDS } from '../../../../../util';
+import useEnableAudienceGroup from '../../../hooks/useEnableAudienceGroup';
+import AudienceErrorModal from './AudienceErrorModal';
+import NotificationWithSVG from '../../../../../googlesitekit/notifications/components/layout/NotificationWithSVG';
+import ActionsCTALinkDismiss from '../../../../../googlesitekit/notifications/components/common/ActionsCTALinkDismiss';
+import BannerGraphicsSVGDesktop from '../../../../../../svg/graphics/audience-segmentation-setup-desktop.svg';
+import BannerGraphicsSVGTablet from '../../../../../../svg/graphics/audience-segmentation-setup-tablet.svg';
+import BannerGraphicsSVGMobile from '../../../../../../svg/graphics/audience-segmentation-setup-mobile.svg';
+import {
+	BREAKPOINT_SMALL,
+	BREAKPOINT_TABLET,
+	useBreakpoint,
+} from '../../../../../hooks/useBreakpoint';
+
+export const AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION =
+	'audience_segmentation_setup_cta-notification';
+
+const breakpointSVGMap = {
+	[ BREAKPOINT_SMALL ]: BannerGraphicsSVGMobile,
+	[ BREAKPOINT_TABLET ]: BannerGraphicsSVGTablet,
+};
+function AudienceSegmentationSetupCTAWidget( { id, Notification } ) {
+	const viewContext = useViewContext();
+	const breakpoint = useBreakpoint();
+	const trackEventCategory = `${ viewContext }_audiences-setup-cta-dashboard`;
+
+	const { invalidateResolution } = useDispatch( CORE_NOTIFICATIONS );
+
+	const { setValues } = useDispatch( CORE_FORMS );
+
+	const showTooltip = useShowTooltip( id );
+	const { isTooltipVisible } = useTooltipState( id );
+
+	const isDismissalFinal = useSelect( ( select ) =>
+		select( CORE_NOTIFICATIONS ).isNotificationDismissalFinal( id )
+	);
+
+	// See TODO note below.
```

Comment:
```suggestion
	// See TODO note below.
```

Can you update this to either:

1. point extract to the relevant TODO or mention which TODO this references with a bit more context?
2. expand on this entirely here?

Even if it's repeating things to do this, the current TODO is without context and I can't immediately see what it references so it's confusing.

Reply:
I removed the inline comment from here, since it's usage is explained in proper place, this is only retrieving a value like any other constant